tests/queries/0_stateless/02716_drop_if_empty.sql

```diff
@@ -0,0 +1,18 @@
+-- Tags: no-parallel
+CREATE TABLE data_02716_1 (v UInt64) ENGINE = MergeTree ORDER BY v;
+CREATE TABLE data_02716_2 (v UInt64) ENGINE = MergeTree ORDER BY v;
+
+CREATE DATABASE db_02716;
```

Comment:
FYI You can avoid `no-parallel` tag by using `{CLICKHOUSE_DATABASE_1:Identifier}` param instead of the database name